engine/runtime/src/main/java/org/enso/interpreter/runtime/error/PanicException.java

```diff
@@ -137,7 +137,7 @@ static UnresolvedSymbol toDisplayText(Object payload, IndirectInvokeMethodNode p
       }
     } catch (Error | Exception e) {
       logger().atError().log("Cannot compute message for " + payload, e);
-      throw UnsupportedMessageException.create(e);
+      throw UnsupportedMessageException.create(e instanceof AbstractTruffleException ? e : null);
```

Comment:
The c2f48f14e95dd8f544c23a7ba1a778cddb6b7226 change should fix [this error](https://github.com/enso-org/enso/actions/runs/11976679725/job/33392873030#step:7:2052) - e.g. regular `UnsupportedMessageException` without any connection to the `e` will be thrown. The actual `e` exception will only be sent to:
```java
logger().atError().log("Cannot compute message for " + payload, e);
```
will we find the stacktrace of the `AssertionError e` in the CI output?